.circleci/windows/package.cfg

```diff
@@ -0,0 +1,79 @@
+[bundle]
+
+packages = {pkg_prefix}libsodium
+
+filename_stub = aeternity-windows
+
+display_name = Aeternity for Windows
+
+description = The full Aeternity node for Windows.
+
+version = VERSION_PLACEHOLDER
```

Comment:
Version is no longer hard-coded and it will be resolved on build time

Reply:
I don't really like the `VERSION_PLACEHOLDER`, perhaps I'd expect `master` or something "real" - but OK

Reply:
it does not really matter what it will be anyway. It does not stop building the package. Also replace will look for `version =`. While missing this step and using it directly it will make this more obvious for debugging